push-notifications-ios/push-notifications-ios/AppDelegate.swift

```diff
@@ -7,18 +7,15 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     let pushNotifications = PushNotifications.shared
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
-        self.pushNotifications.start(instanceId: "f918950d-476d-4649-b38e-6cc8d30e0827")
+        self.pushNotifications.start(instanceId: "YOUR_INSTANCE_ID") // Can be found here: https://dash.pusher.com
```

Comment:
What's the motivation for this change?

Reply:
To be consistent with the other SDKs and [docs](https://docs.pusher.com/beams/reference/ios#quickstart).